homeworks/taras.bobov_LotusElize/GamesProject/css/games.css

```diff
@@ -0,0 +1,92 @@
+ul,
+li,
+h2,
+p {
+  margin: 0;
+}
+
+.content_game-cards {
+  width: 1175px;
+  height: 545px;
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-template-rows: repeat(2, 1fr);
+  gap: 25px;
+  margin-bottom: 146px;
+  margin-top: 40px;
+}
+
+.card {
+  width: 275px;
+  height: 260px;
+  background-color: #ECECEC;
+  color: #221F1F;
+  display: grid;
+  padding: 12px 12px 22px;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  border-radius: 12px;
+  -webkit-border-radius: 12px;
```

Comment:
Border-radius is well supported, so you can remove all these vendor prefixes (`-webkit-border-radius`...`-o-border-radius`)
Please fix in the next task